modules/virt-example-vf-host-services.adoc

```diff
@@ -59,12 +56,11 @@ spec:
 <9> Identifies the VF with an ID of `0`.
 <10> Sets a maximum transmission rate, in Mbps, for the VF. This sample value sets a rate of 200 Mbps.
 
-The following YAML file is an example of a manifest that creates a VLAN interface on top of a VF and adds it to a bonded network interface.
-It includes samples values that you must replace with your own information.
+The following YAML file is an example of a manifest that adds a VF for a network interface. In this configuration, the `ens1f1v0` VF is created on the `ens1f1` physical interface, and this VF is added to a bonded network interface `bond0`. The bond uses `active-backup` mode for redundancy. In this example, the VF is configured to use hardware offloading to manage the VLAN directly on the physical interface.
+This YAML includes samples values that you must replace with your own information.
```

Comment:
It isn't clear what these values are because there aren't any <replace_with_your_value> in the example. I recommend actually moving a lot of this to comments in the YAML file.  For example, the first second sentence on line 59 could be moved to commented out material on line 74 and 84.